Kickstarter-iOS/Views/Controllers/BetaToolsViewController.swift

```diff
@@ -149,6 +154,11 @@ internal final class BetaToolsViewController: UIViewController {
       UIAlertAction.init(title: "Cancel", style: .cancel)
     )
 
+    if UIDevice.current.userInterfaceIdiom == .pad {
```

Comment:
Would it make sense to extract these checks to a `UIAlertController`’s extension in order to prevent this from happening in the real code base and be more DRY? We can either have a convenience function to present alert or just to create new instance

Reply:
That's a good idea. But instead of creating a `UIAlertController`'s extension, what about create a `UIViewController`'s extension to present the alert? We can add a function containing that logic like:
```
public func present(_ alert: UIAlertController, sourceView: UIView? = nil) 
```
And, from the ViewController, simply call 
```
self.present(alert, sourceView: self.languageSwitcher)
```

Reply:
Nm. I just extracted the logic to the `UIAlertController` extension.